models/cow_protocol/ethereum/cow_protocol_ethereum_batches.sql

```diff
@@ -1,6 +1,7 @@
 {{  config(
         alias='batches',
         materialized='incremental',
+        partition_by = ['block_date'],
```

Comment:
In `cow_protocol_ethereum_batches.sql`, block_date is neither present in the `batch_values` CTE nor the `combined_batch_info` one